Story/Fireisland/CoreFireisland.cs

```diff
@@ -0,0 +1,240 @@
+        Story.KillQuest(4077, "Pyrewatch", new[] { "Lavazard", "Living Lava" });
+        Story.MapItemQuest(4077, "Pyrewatch", 3161, 5);
+
+        //Defend Pyrewatch Peak 4078
+        Story.KillQuest(4078, "Pyrewatch", "Storm Scout");
+
+        //Signal Fire 4079
+        Story.KillQuest( 4079, "Pyrewatch", new[] { "Storm Scout", "Fire Pikeman", "Flame Soldier", "Fyreborn Tiger" } );
+
+        //Spreading Like Wildfire 4080
+        Story.MapItemQuest(4080, "Pyrewatch", 3162, 4);
+
+        //Pyrewatch Defender Badge 4081
+        Story.KillQuest( 4081, "Pyrewatch", new[] { "Fire Pikeman", "Firestorm Knight", "Flame Soldier", "Storm Scout" } );
+    }
+
+    public void Fireforge()
+    {
+        if (Core.isCompletedBefore(4226))
+        {Core.Logger("You have already completed Fireforge storyline");
+            return;}
+
+        Story.PreLoad();
+
+        //Round 1: Firestorm Scouts 4216
+        Story.KillQuest(4216, "Fireforge", "Firestorm Scout");
+
+        //Round 2: Firestorm Soldiers 4217
+        Story.KillQuest(4217, "Fireforge", "Firestorm Soldier");
+
+        //Round 3: Firestorm Pikemen 4218
+        Story.KillQuest(4218, "Fireforge", "Fire Pikeman");
+
+        //Round 4: Firestorm Knights 4219
+        Story.KillQuest(4219, "Fireforge", "Firestorm Knight");
+
+        //Round 5: Firestorm Corporals 4220
+        Story.KillQuest(4220, "Fireforge", "Firestorm Corporal");
+
+        //Round 6: Firestorm Tigers 4221
+        Story.KillQuest(4221, "Fireforge", "Armored Tiger|Firestorm Tiger");
+
+        //Round 7: Firestorm Cavalry 4222
+        Story.KillQuest(4222, "Fireforge", "Tiger Cavalry");
+
+        //Round 8: Firestorm Majors 4223
+        Story.KillQuest(4223, "Fireforge", "Firestorm Major");
+
+        //Round 9: Firestorm Blazebinders 4224
+        Story.KillQuest(4224, "Fireforge", "Blazebinder");
+
+        //Round 10: Firestorm General 4225
+        Story.KillQuest(4225, "Fireforge", "Firestorm General");
+
+        //Bonus Round: Flamewing the Bright 4226
+        Story.KillQuest(4226, "Fireforge", "Flamewing");
+
+        //Final Round: Tyndarius Tigermaster 4230
+        Core.EquipClass(ClassType.Solo);
+        Story.KillQuest(4230, "Fireforge", "Tyndarius");
+    }
+
+    public void Lavarun()
+    {
+        if (Core.isCompletedBefore(4235))
+        {Core.Logger("You have already completed Lavarun storyline");
+            return;}
+        Story.PreLoad();
+        //Defeat Phedra 4231
+        Core.EquipClass(ClassType.Solo);
+        Story.KillQuest(4231, "Lavarun", "Phedra");
+
+        //Defeat Mega Tyndarius 4232
+        Core.EquipClass(ClassType.Solo);
+        Story.KillQuest(4232, "Lavarun", "Mega Tyndarius");
+
+        //The Onslaught Fights On 4235
+        Story.KillQuest(4235, "Lavarun", "Firestorm Soldier|Firestorm Scout");
+    }
+
+    public void Brimstone()
+    {
+        if (Core.isCompletedBefore(4115))
+        {Core.Logger("You have already completed Brimstone storyline");
```

Comment:
make it seperate 
{
Core.Logger("You have already completed Brimstone storyline");
return;
}